opt_PXL.ino

```diff
@@ -38,7 +39,7 @@ int cycleImageCount= 0;
 byte x;
 
 //OLED details
-#define OLED_RESET 4
+#define OLED_RESET -1

```

Comment:
What does this `#define` change mean?

Reply:
The `OLED_RESET 4` change is what is suggested by the ADAFRUIT gods. Something about stopping the OLED pin being used as a reset causing weird display results? I don't think I've noticed a difference between the two? Maybe more testing needed.